install_api.sh

```diff
@@ -333,6 +333,13 @@ setup_api_permissions () {
 
     # API users permissions
     exec_cmd "chmod 660 $API_PATH/configuration/auth/user"
+
+    # permissions for execute verify-agent-conf binary
```

Comment:
This must be done in the makefile / install.sh of Wazuh.

Reply:
ok!